src/com/cft/contactmerge/MapMaker.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * MapMaker is responsible for creating the map file that tells ContactImporter
+ * and MergeFileExporter how to map properties (FirstName, Address, etc) to
+ * the columns in the file.
+ */
+package com.cft.contactmerge;
+
+import java.util.*;
+
+public class MapMaker {
+    public void addColumnMap(String property, int columnInFile)
+    {
```

Comment:
for addColumnMap and removeColumnMap... we have "String property" and "String PropertyName". Is there a difference, or should we be consistent in the naming structure?

Reply:
Thanks. Updated.